app/assets/stylesheets/components/_option-select.scss

```diff
@@ -3,6 +3,15 @@
   padding: 5px;
   margin-bottom: $gutter;
 
+  .govuk-hint {
+    position: absolute;
+    left: -9999em;
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
```

Comment:
I'm not able to see the hint in the demo app (with and without JS). Do you know where I can find it? I can see it in the DOM, though! :)

```
<span id="checkboxes-4259dd0a-hint" class="govuk-hint">Select all that apply.</span>
```

Reply:
Yes - is this intentionally being hidden?

Reply:
Is there a reason why you chose this method and not applying the `visuallyhidden` class?

Reply:
Yeah, I can't put that class on that element because it's part of the component. I wanted to use a mixin but the govuk one didn't work. I thought it was easiest to put it in longhand and hope that someone would point out a solution :)

Reply:
Good point, didn't spot that. The only solutions I can think of are 1) making it an option in the component, but I'm not sure how often this use case will come up; or 2) putting a separate hint outside of the component, perhaps where we already indicate '[x] selected' when people select checkboxes? But not sure if moving it makes it less useful.

Reply:
I thought about making it an option in the component but if removed it causes an axe accessibility error. But the design for the finders doesn't have it in, so I decided the best thing to do was this i.e. hide it except from screen readers.

These things are going to be (mostly) replaced by accessible autocompletes shortly anyway.

Reply:
This PR https://github.com/alphagov/govuk_publishing_components/pull/684 updates the checkboxes component so that we can hide the hint text using options, so once that's merged I'll modify this PR to use it accordingly, should hopefully deal with this issue.